src/analysis/complex/abs_max.lean

```diff
@@ -0,0 +1,242 @@
+/-
+Copyright (c) 2022 Yury G. Kudryashov. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yury G. Kudryashov
+-/
+import analysis.complex.cauchy_integral
+import analysis.convex.integral
+import analysis.normed_space.completion
+import topology.algebra.order.extr_closure
+
+/-!
+# Maximum modulus principle
+
+In this file we prove several versions of the maximum modulus principle and Liouville's Theorem.
```

Comment:
```suggestion
In this file we prove several versions of the maximum modulus principle.
```

Reply:
Indeed, I splitted this into two files and forgot to fix the module docstring.